plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1362,7 +1362,7 @@ private boolean checkOvsNetwork(final String networkName) {
     }
 
     public boolean passCmdLine(final String vmName, final String cmdLine) throws InternalErrorException {
-        final Script command = new Script(_patchScriptPath, 30 * 1000, s_logger);
+        final Script command = new Script(_patchScriptPath, 300 * 1000, s_logger);
```

Comment:
Curious question - Is that a desired timeout value? Seems too high.

Reply:
In environment with slow cpu and storage network, the timeout may be even higher. The logic of 5minute timeout comes from the libvirt wrapper command handler:
```
                // try to patch and SSH into the systemvm for up to 5 minutes
                for (int count = 0; count < 10; count++) {
```
For historic reasons, you can see CLOUDSTACK-2823

Reply:
Thanks!